packages/tinacms/src/toolkit/fields/plugins/mdx-field-plugin/plate/components/plate-ui/image-toolbar-button.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { withRef } from '@udecode/cn'
+
+import { Icons } from './icons'
+
+import { ToolbarButton } from './toolbar'
+import { useEditorState } from '@udecode/plate-common'
+import { helpers } from '../../plugins/core/common'
+import { ELEMENT_IMG, insertImg } from '../../plugins/create-img-plugin'
+import { useCMS } from '@toolkit/react-core'
```

Comment:
**nitpick**: Could the imports be ordered to make it clear which ones are external imports?

Reply:
Will be handled in a separate PR